app/models/comment.rb

```diff
@@ -1,4 +1,11 @@
 class Comment < ApplicationRecord
-  belongs_to :author
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
- hey, this is how to reference the user table: ` belongs_to :author, class_name: 'User'` Update this. Also, you didn't run any migration, try and update your migration.

Reply:
fixed